src/backup_camera.cpp

```diff
@@ -126,11 +138,28 @@ bool BackupCamera::process_events()
                         signalToQuit();
                         return false;
                         break;
+
+                    case SDLK_f:
```

Comment:
For this pull request, `SDLK_f` is fine. However, I believe @enochtsang wants a certain module pulling in inputs from the system somehow... using a stream? You'd have to ask him. It will probably be the next PR you'll work on